src/tests/__snapshots__/EtaxForm.test.js.snap

```diff
@@ -0,0 +1,333 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `
+<div
+  className="row col-12"
+>
+  <div
+    className="col-4"
+  >
+    <div
+      aria-describedby="houseNo-helper-text"
+      className="MuiFormControl-root-1 MuiFormControl-marginNormal-2 col-12"
```

Comment:
Just a little heads-up: There used to be an issue where the class names generated by material-ui were not entirely deterministic, so there were times when tests will fail due to library updates or addition of new styles.

While I doubt this still occurs, if you happen to encounter any issue, this issue might be able to help you: https://github.com/mui-org/material-ui/issues/9492

Reply:
@phoomparin เออหว่ะ ดูผิด 555555 เช เช เช

Reply:
@phoomparin @champillon  รบกวน https://github.com/it-kmitl-2018/group3-frontend/pull/9 อันนี้ด้วยนะครับ (-/\-)